packages/riverpod_analyzer_utils/lib/src/riverpod_element.dart

```diff
@@ -11,6 +11,7 @@ class RiverpodAnnotationElement {
   @internal
   RiverpodAnnotationElement({
```

Comment:
Changes in this file needs to be tested. Tests are located in a different folder within `packages` with a similar name,

Reply:
Generator changes also need to be tested.


And you'd also need to test that this new feature works well with riverpod_lint (specifically for lints like `provider_dependencies`)